migrations/20220810211254_drop_column_and_fks.down.fizz

```diff
@@ -0,0 +1,7 @@
+add_foreign_key("requirement_types", "department_id", {"departments": ["id"]}, {
+    "departmentFK": "departmentFK",
+    "on_delete": "cascade",
+    "on_update": "cascade",
+})
+change_column("requirement_types", "department_id", "uuid", {})
+add_column("requirement_types", "department_id", "uuid" {})
```

Comment:
Please try running this migration and fix it. The order of the statements is not the correct.